CHANGES

```diff
@@ -110,6 +101,11 @@ Notations
 
 Tools
 
+- New warnings system for coqtop/coqc. Can be configured with the flag "-w" or
+  with the Vernacular command "Set Warnings".
+- Warnings are now printed by coqc.
+- In asynchronous mode, Coq is now capable of recovering from errors and
+  continue processing the document.
 - coqc accepts a -o option to specify the output file name
```

Comment:
For what of changes you mean? These are changes that users can see, not really internal, right?